Formula/borgbackup.rb

```diff
@@ -0,0 +1,32 @@
+class Borgbackup < Formula
+  include Language::Python::Virtualenv
+
+  desc "Deduplicating archiver with compression and authenticated encryption"
+  homepage "https://borgbackup.readthedocs.io/en/stable/"
+  url "https://github.com/borgbackup/borg/releases/download/1.1.13/borgbackup-1.1.13.tar.gz"
+  sha256 "164a8666a61071ce2fa6c60627c7646f12e3a8e74cd38f046be72f5ea91b3821"
+  license "BSD-3-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
+  depends_on :osxfuse
+  depends_on "python@3.8"
```

Comment:
From setup.py (1.1-maint branch, master is rather different):
```
# True: use the shared liblz4 (>= 1.7.0 / r129) from the system, False: use the bundled lz4 code
prefer_system_liblz4 = True

# True: use the shared libzstd (>= 1.3.0) from the system, False: use the bundled zstd code
prefer_system_libzstd = True

# True: use the shared libb2 from the system, False: use the bundled blake2 code
prefer_system_libb2 = True
```

Guess these 3 need checking.

If the idea is to use the system library, the version requirements needs to be satisfied.

If not, maybe patch setup.py so it uses the bundled stuff.

Reply:
Added those 3 libraries as dependencies.

Reply:
Since the formula uses a specific version of Borg (1.1.13), compatibility with 1.1-maint shouldn't be an issue.

Reply:
Looks like it uses the Brew-provided libs:

```
$ lsof -c python3.8 | grep 'bb2\|zstd\|lz4'
python3.8 31721 manu  txt     REG                1,4   105784            28741423 /usr/local/Cellar/libb2/0.98.1/lib/libb2.1.dylib
python3.8 31721 manu  txt     REG                1,4   139704            28741498 /usr/local/Cellar/lz4/1.9.2/lib/liblz4.1.9.2.dylib
python3.8 31721 manu  txt     REG                1,4   667796            28741605 /usr/local/Cellar/zstd/1.4.5/lib/libzstd.1.4.5.dylib
```

Reply:
Anything else to change from your side, @ThomasWaldmann?

Reply:
libs look ok.